# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/EventMessageList.h
# Commit: f8a2a169b296
# Full Hash: f8a2a169b2969d29e04733ae27a56459e5359973
# Author: Alexandru Marc <amarc@mozila.com>
# Date: 2025-05-16 09:19:19
# Regressor Bug: 1550462
# File Overlap Count: 1
# Description:
#   Revert "Bug 1550462 - part 3: Enable `pointerrawupdate` event r=smaug" for causing mochitest failures @ test_pointerrawupdate_event_count.html
#   
#   This reverts commit eb27445e0352d6b4df540adb40e21f72fc052e80.
#   
#   Revert "Bug 1550462 - part 2: Make `PresShell::HandleEvent` dispatch preceding `pointerrawupdate` event r=smaug,dom-core,edgar"
# ==============================================================================

diff -r 8d25773fc82f -r f8a2a169b296 widget/EventMessageList.h
--- a/widget/EventMessageList.h	Fri May 16 06:20:47 2025 +0000
+++ b/widget/EventMessageList.h	Fri May 16 09:30:09 2025 +0300
@@ -93,14 +93,6 @@
 NS_EVENT_MESSAGE(eMouseLeave)
 NS_EVENT_MESSAGE(eMouseTouchDrag)
 NS_EVENT_MESSAGE(eMouseLongTap)
-// eMouseRawUpdate is for dispatching ePointerRawUpdate caused by a mouse input.
-// When we dispatch ePointerRawUpdate, we need to handle the steps defined by
-// the spec in "fire a pointer event" section like the other pointer event.
-// The steps are handled by
-// PresShell::EventHandler::DispatchPrecedingPointerEvent() and for using it, we
-// should dispatch this internal event instead of dispatching ePointerRawUpdate
-// directly.
-NS_EVENT_MESSAGE(eMouseRawUpdate)
 NS_EVENT_MESSAGE(eMouseExploreByTouch)
 NS_EVENT_MESSAGE_FIRST_LAST(eMouseEvent, eMouseMove, eMouseExploreByTouch)
 
@@ -108,13 +100,6 @@
 NS_EVENT_MESSAGE(ePointerAuxClick)
 
 // Pointer spec events
-// NOTE: We handle the steps before dispatching a pointer event which is defined
-// by the spec in "fire a pointer event" section in
-// PresShell::EventHandler::DispatchPrecedingPointerEvent().  Therefore,
-// we should not dispatch ePointer* events with
-// `PresShell::EventHandler::HandleEvent` directly.  Create a new internal
-// message for implementing a new pointer event if the new event is defined as
-// dispatched with "fire a pointer event" steps.
 NS_EVENT_MESSAGE(ePointerMove)
 NS_EVENT_MESSAGE(ePointerUp)
 NS_EVENT_MESSAGE(ePointerDown)
@@ -123,7 +108,6 @@
 NS_EVENT_MESSAGE(ePointerEnter)
 NS_EVENT_MESSAGE(ePointerLeave)
 NS_EVENT_MESSAGE(ePointerCancel)
-NS_EVENT_MESSAGE(ePointerRawUpdate)
 NS_EVENT_MESSAGE(ePointerGotCapture)
 NS_EVENT_MESSAGE(ePointerLostCapture)
 NS_EVENT_MESSAGE_FIRST_LAST(ePointerEvent, ePointerMove, ePointerLostCapture)
@@ -448,14 +432,6 @@
 NS_EVENT_MESSAGE(eTouchEnd)
 NS_EVENT_MESSAGE(eTouchCancel)
 NS_EVENT_MESSAGE(eTouchPointerCancel)
-// eTouchRawUpdate is for dispatching ePointerRawUpdate caused by a touch.
-// When we dispatch ePointerRawUpdate, we need to handle the steps defined by
-// the spec in "fire a pointer event" section like the other pointer event.
-// The steps are handled by
-// PresShell::EventHandler::DispatchPrecedingPointerEvent() and for using it, we
-// should dispatch this internal event instead of dispatching ePointerRawUpdate
-// directly.
-NS_EVENT_MESSAGE(eTouchRawUpdate)
 
 // Pointerlock DOM API
 NS_EVENT_MESSAGE(ePointerLockChange)